docs/theming.md

```diff
@@ -110,16 +110,15 @@ Here's the markup:
           <h2 class="mdc-card__subtitle">Go with the flow.</h2>
         </section>
         <section class="mdc-card__actions">
-          <button class="mdc-button mdc-button--primary mdc-button--compact mdc-card__action">Pick this element</button>
+          <button class="mdc-button mdc-button--compact mdc-card__action">Pick this element</button>
         </section>
       </div>
     </div>
   </body>
 </html>
 ```
 
-You'll see that we have a number of pretty empty looking cards, with black text on a white background. The only hint of
-color comes from the buttons, which we've made use the primary color by adding the `mdc-button--primary` class.
+You'll see that we have a number of pretty empty looking cards, with black text on a white background. For buttons, they adopt the baseline color by default.
```

Comment:
Maybe we should change this to use the MDC Fab, which defaults to primary color?

Reply:
Sounds good, that seems better meet the purpose here actually.